File: JavaScript Excel Examples/RangesApiDemo/setRangeColumnWidth.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.RangesApi(apiClient);
    
    var name = "setRangeColumnWidth.xlsx";
    var sheetName = "Sheet1";
    var opts = {folder: "input" };
    var range = new SpirecloudExcel.Range();
    range.FirstRow = 1;
    range.FirstColumn = 1;
    range.ColumnCount = 5;
    range.RowCount = 2;
    var value = 15;
    instance.setRangeColumnWidth(name, sheetName, range, value, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();

File: Python Excel Examples/RangesApiDemo/setRangeColumnWidth.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.ranges_api.RangesApi(configuration)

name = "SetRangeColumnWidth.xlsx"
folder = "Ranges"
storage = ""
sheetName = "Sheet1"
Range = spirecloudexcel.models.Range()
Range.first_row = 1
Range.first_column = 1
Range.row_count = 2
Range.column_count = 5
value = 15
api.set_range_column_width(name, sheetName, Range, value, storage = storage, folder = folder)